### PR TITLE
Develop2.x rmb zoom

### PR DESCRIPTION
**Pull request by todderasesareddot**
Enable Alt-RMB graph zoom like Canvas and Maya Node editor.  Make graph zoom centered around mouse.

### CODE REVIEW

**Comment by EricTRocks** [comment]
@phtaylor Can you review the changes to Pyflowgraph? Will these interfere with other apps that are building on top of it?

**Comment by todderasesareddot** [comment]
Shouldn't affect anything else. Overall a pretty small change.  Manip numbers are the same for existing modes.

**Comment by EricTRocks** [comment]
I have a glitch when zooming really close. There is an odd shift that comes and goes. Towards the end of this gif:

rightclickzoomglitch

**Comment by todderasesareddot** [comment]
Hmmm... yeah totally.  Once it hits a certain zoom level, it glitches.  But it works fine otherwise.  I wonder if the QGraphicsView auto-align is doing something strange when it scales to a certain amount.  I'm not too familiar with it.  Phil probably has more insight.

File: Python/kraken/ui/GraphView/kgraph_view.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 from PySide import QtGui, QtCore
 
-from pyflowgraph.graph_view import GraphView
+from pyflowgraph.graph_view import *
```
Comment (EricTRocks) [inline Python/kraken/ui/GraphView/kgraph_view.py]: Should really only import the objects you need from modules and not * Other module level variables in that module may conflict down the line if more are added.

`from pyflowgraph.graph_view import MANIP_MODE_ZOOM, MANIP_MODE_SELECT, MANIP_MODE_PAN`

**Comment by EricTRocks** [comment]
Just tested on the main develop branch and this issue is there too. That zoom glitch wasn't introduced with this addition.

**Comment by todderasesareddot** [comment]
Yeah, I have a feeling the zoom glitch has something to do with the auto-centering of the scene.  When you resize the view, the scene auto-centers, but only until a certain zoom level (or maybe scene rect range).

**Pull request merged**